docs/guides/external-node/08_pruning.md

```diff
@@ -0,0 +1,38 @@
+# Pruning
+
+It is possible to configure Node to periodically remove all data from batches older than a threshold, both from Postgres
+and from tree.
+
+> [!NOTE]
+>
+> If you need a node with data retention period of up to a few days, please set up a node from a snapshot (see previous
+> chapter) and wait for it to have enough data. Pruning an archival node can take unpractical amount of time. In the
+> future we will be offering pre-pruned DB snapshots with a few months of data.
+
+You can enable pruning by setting
+
+```
+EN_PRUNING_ENABLED=true
+```
+
+By default, it will keep history for 7 days. You can configure retention period using:
+
+```
+EN_PRUNING_DATA_RETENTION_SEC: '259200' // 3 days
```

Comment:
Nit: Please use shell formatting for consistency here:

```shell
EN_PRUNING_DATA_RETENTION_SEC=259200 # 3 days
```

Reply:
For me, shell should be used in places where we put a runnable command. For instance my IDE puts "run" icon on the left of such block, imo it looks weird in this case

Reply:
I had better idea, I used yaml, now it even colors nicely in my IDE